react/PartnerSites/Products/Products.js

```diff
@@ -79,7 +98,12 @@ export default function PartnerSites({ locale, linkRenderer }) {
   );
 }
 
-PartnerSites.propTypes = {
-  locale: PropTypes.string.isRequired,
-  linkRenderer: PropTypes.func.isRequired
+Products.propTypes = {
+  locale: PropTypes.oneOf(['AU', 'NZ']).isRequired,
+  linkRenderer: PropTypes.func.isRequired,
+  activeProduct: PropTypes.oneOf(['Jobs', 'Courses', 'Businesses for sale', 'Volunteering'])
+};
+
+Products.defaultProps = {
+  activeProduct: null
```

Comment:
@markdalgleish It's currently set to null because we don't set an active one on https://talent.seek.com.au.

If this is just an over-site and `activeProduct` should be set to `Jobs` regardless whether you're on the Candidate or Employer "Jobs" site, then I'm happy to set default to `Jobs`.

Also, I'm not sure if you're aware but there are talks going on at the moment to include Employer in this list as it's own nav item. Not sure if that impacts setting a default `activeProduct` now or if we just refactor this Component if/when this new item comes into the mix.

Reply:
Yep, makes sense—let's not include a default 👍